runtime/src/iree/base/loop_emscripten.js

```diff
@@ -49,9 +51,59 @@ const IreeLibraryLoopEmscripten = {
       }
     }
 
+    class LoopCommandWaitOne extends LoopCommand {
+      constructor(
+          scope, operationId, callback, user_data, timeout_ms, wait_promise,
+          loop) {
+        super();
+
+        this.callback = callback;
+        this.user_data = user_data;
+        this.loop = loop;
+        this.abortFn = undefined;
```

Comment:
I'm undecided on code style here. The C APIs use `userscore_case`, while JS typically uses `camelCase`. I tried to keep the same case across the boundary or when a function is mostly one or the other, but some places like this have a weird mix.

Reply:
I'd normally prefer all JS, though being able to find-replace within function signatures across C and JS is useful.

Reply:
Just changed to all JS. There aren't as many functions now, so lining them up isn't too difficult.